src/metacommands.cpp

```diff
@@ -393,7 +393,7 @@ static std::map<string, function<Attribute*(string)>> name2constructor {
 Attribute* MetaCommands::newAttributeWithType(string typestr, string attr_name, Output output) {
     auto it = name2constructor.find(typestr);
     if (it == name2constructor.end()) {
-        output << "error: unknown type \"" << typestr << "\"";
+        output.perror() << "error: unknown type \"" << typestr << "\"";
```

Comment:
"error:" can probably be removed?

Reply:
(Which probably is the reason for the merge conflict)

Reply:
Indeed, thanks! (it's resolved now)

Reply:
Thanks!